docs/configuration/configuration.md

```diff
@@ -317,11 +317,27 @@ tls_config:
 
 # A list of services for which targets are retrieved. If omitted, all services
 # are scraped.
+# See https://www.consul.io/api/catalog.html#list-nodes-for-service to know more
```

Comment:
This goes with node_meta

Reply:
This goes to both `tag` and `node_meta`, but moved after `sevices`